vue/components/organisms/pickers/pickers.vue

```diff
@@ -471,6 +477,10 @@ export const Pickers = {
             type: Boolean,
             default: false
         },
+        enableCenterScroll: {

```

Comment:
Maka this a `String` so that we can support multiple strategies. The default can be `left`.